### PR TITLE
Parallel rendering

### PR DESCRIPTION
**Pull request by doudou**
Changes in preparation for rendering multiple poses in parallel

### CODE REVIEW

File: src/NormalDepthMap.cpp [hunk 1]
```diff
@@ -133,6 +163,18 @@ osg::ref_ptr<osg::Group> NormalDepthMap::createTheNormalDepthMapShaderNode(
     osg::ref_ptr<osg::Uniform> drawDepthUniform(new osg::Uniform("drawDepth", drawDepth));
     ss->addUniform(drawDepthUniform);
 
+    osg::Matrixf viewMatrix(osg::Matrixf::identity());
+    osg::ref_ptr<osg::Uniform> viewMatrixUniform(new osg::Uniform("viewMatrix", viewMatrix));
+    ss->addUniform(viewMatrixUniform);
+
+    double aspect_ratio = tan(maxHorizontalAngle * 0.5) / tan(maxVerticalAngle * 0.5);
```
Comment (alcantara09) [inline src/NormalDepthMap.cpp]: setFOV(maxHorizontalAngle, maxVerticalAngle); ??

Comment (doudou) [reply src/NormalDepthMap.cpp]: I thought about doing so, but setFOV requires the _normalDepthMapNode to be set, which is done after this method. This was leading to a half-initialized created node and "please don't forget to call setFOV afterwards". So, that's how I ended up doing it.

Comment (doudou) [reply src/NormalDepthMap.cpp]: I'll factor out the complicated code, this would improve.

Comment (doudou) [reply src/NormalDepthMap.cpp]: Done

**Pull request merged**